app/components/UserWidget/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { createStructuredSelector } from 'reselect';
+import { removeToken, loggedIn, loggedOut } from 'components/AuthUser/lib';
+import { logout } from 'components/AuthUser/actions';
+import { selectGivenName, selectPicture } from 'components/AuthUser/selectors';
+import UserBadge from 'components/UserBadge';
+import styles from './styles.css';
+import buttonStyles from 'components/Button/styles.css';
+
+class UserWidget extends React.Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
I think this is a smart components connected to state, same almost goes for `AuthUser` with reducers and actions. Are they supposed to be in `components` folder?